.github/workflows/build.yml

```diff
@@ -10,7 +10,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ "stable", "latest" ]
+        target:
+          - "latest"
```

Comment:
We could just remove `matrix.target` entirely (and 2x below)

Reply:
I'm kind of in favor of leaving breadcrumbs in place in case we need multiple branches in the future.